packages/storage/src/implementation/metadata.ts

```diff
@@ -206,20 +207,23 @@ export function toResourceString(
   return JSON.stringify(resource);
 }
 
-export function metadataValidator(p: unknown): void {
-  if (!type.isObject(p) || !p) {
-    throw 'Expected Metadata object.';
-  }
+export function validateMetadata(p: { [key: string]: unknown }): void {
```

Comment:
I actually got rid of this entirely in storage-exp because it's checking types of Metadata fields which should be caught by typescript check? Or did I make a mistake?